viz/charts/linkerd-viz/README.md

```diff
@@ -149,6 +150,7 @@ Kubernetes: `>=1.22.0-0`
 | prometheus.podAnnotations | object | `{}` | annotations for the prometheus pod |
 | prometheus.proxy | string | `nil` |  |
 | prometheus.remoteWrite | string | `nil` | Allows transparently sending samples to an endpoint. Mostly used for long term storage. |
+| prometheus.resources | object | `{"cpu":{"limit":null,"request":null},"ephemeral-storage":{"limit":"","request":""},"memory":{"limit":null,"request":null}}` | Prometheus data volume size.  size: |
```

Comment:
Ah yes, I feel like this should be a separate PR? or maybe this is just included by mistake.